obd/obd.py

```diff
@@ -155,40 +155,46 @@ def status(self):
             return self.interface.status()
 
 
-    # not sure how useful this would be
-
-    # def ecus(self):
-    #     """ returns a list of ECUs in the vehicle """
-    #     if self.interface is None:
-    #         return []
-    #     else:
-    #         return self.interface.ecus()
+    def get_ecus(self):
+        """ returns a list of ECUs in the vehicle """
+        if self.interface is None:
+            return []
+        else:
+            return self.interface.get_ecus()
```

Comment:
Do you have a specific use for this function? I couldn't quite justify yet, since we only really label the tx ID of the engine.